FinalProject/gs-rest-service-master/complete/src/main/java/dao/CartDao.java

```diff
@@ -0,0 +1,21 @@
+package dao;
+
+import java.sql.SQLException;
+import java.util.List;
+
+import bo.Cart;
+import bo.CartItem;
+import bo.User;
+
+public interface CartDao {

```

Comment:
Minor: it's not necessary to set access modifiers in method here because are always public in an interface.